src/modules/transaction/components/ExportCSVComponent.vue

```diff
@@ -32,18 +32,42 @@
               <div>{{ accPublicKey }}</div>
             </div>
           </div>
-          <div class="py-3">Please enter a range:</div>
+          <div class="py-3">Choose download option:</div>
           <div class="flex">
-            <div class="mr-4">
-              <label for="startBlock" class="font-bold block mb-2"> Start Block</label>
-              <div class="border border-gray-200 px-2 py-1 focus:outline-none">
-                <input type="number" v-model="startBlock" class="focus:outline-none" placeholder="0" />
-              </div>
+            <div class="mr-3">
+              <input type="radio" id="blockNumber" value="blockNumber" class="mr-1" v-model="option" />
+              <label for="blockNumber">Block Number</label>
             </div>
             <div>
-              <label for="endBlock" class="font-bold block mb-2"> End Block </label>
-              <div class="border border-gray-200 px-2 py-1 focus:outline-none">
-                <input type="number" v-model="endBlock" class="focus:outline-none" placeholder="0" />
+              <input type="radio" id="day" value="day" class="mr-1" v-model="option" />
+              <label for="date">Day</label>
+            </div>
+          </div>
+          <div v-if="option === 'blockNumber'">
+            <div class="py-3">Please enter a range:</div>
+            <div class="flex">
+              <div class="mr-4">
+                <label for="startBlock" class="font-bold block mb-2"> Start Block</label>
+                <div class="border border-gray-200 px-2 py-1 focus:outline-none">
+                  <input type="number" v-model="startBlock" class="focus:outline-none" placeholder="0" />
+                </div>
+              </div>
+              <div>
+                <label for="endBlock" class="font-bold block mb-2"> End Block </label>
+                <div class="border border-gray-200 px-2 py-1 focus:outline-none">
+                  <input type="number" v-model="endBlock" class="focus:outline-none" placeholder="0" />
+                </div>
+              </div>
+            </div>
+          </div>
+          <div v-else>
+            <div class="py-3">Please enter the number of days:</div>
+            <div class="flex justify-center items-center">
+              <div class="mr-4">
+                <label for="startBlock" class="font-bold block mb-2">Days</label>
+                <div class="border border-gray-200 px-2 py-1 focus:outline-none">
+                  <input type="number" v-model="startBlock" class="focus:outline-none" placeholder="0" />
+                </div>
```

Comment:
please add in `Latest` or `Last` so user to avoid confusion. And please use another variable for the `days`